src/components/Tile/Tile.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import Tile from './Tile';
+
+describe('Tile component', () => {
+  it('should render tile with name and url', () => {
+  const component = mount(<Tile name="test" url="www.test.com" />, {context: { insertCss: () => {} }});
+  expect(component.text()).toEqual('test');
+  expect(component.find('Link').prop("to")).toEqual('www.test.com')
+  });
+
+  it('should render tile with null values', () => {
```

Comment:
Why? What is the point of an empty div with nothing in it?

Reply:
It is testing that the default props are getting passed. Though these are required props so we should not have to use the default values ever